scripts/build.py

```diff
@@ -354,16 +452,29 @@ def build_third_party_libs():
 
     print 'Building third party libs...'
 
-    third_party_js = os.path.join(
+    DEV_THIRD_PARTY_JS = os.path.join(
         THIRD_PARTY_GENERATED_DEV_DIR, 'js', 'third_party.js')
-    third_party_css = os.path.join(
+    DEV_THIRD_PARTY_CSS = os.path.join(
         THIRD_PARTY_GENERATED_DEV_DIR, 'css', 'third_party.css')
-    fonts_dir = os.path.join(THIRD_PARTY_GENERATED_DEV_DIR, 'fonts', '')
+    DEV_FONTS_DIR = os.path.join(THIRD_PARTY_GENERATED_DEV_DIR, 'fonts', '')
 
     dependency_filepaths = get_dependencies_filepaths()
-    _join_files(dependency_filepaths['js'], third_party_js)
-    _join_files(dependency_filepaths['css'], third_party_css)
-    _copy_fonts(dependency_filepaths['fonts'], fonts_dir)
+    if not does_directory_tree_exist(DEV_THIRD_PARTY_JS):
+        os.makedirs(os.path.dirname(DEV_THIRD_PARTY_JS))
```

Comment:
Why do you need to make a directory for something that's not a directory?

Reply:
Using `_ensure_directory_exist()` in favor of using repeated if block